README.md

```diff
@@ -47,59 +43,55 @@ storage-component.googleapis.com
 ```bash
 python3 -m venv venv
 source ./venv/bin/activate
-pip install ml-pipeline-gen
+pip3 install ml-pipeline-gen
 ```
 
-### Kubeflow
-Create a Kubeflow deployment using Cloud Marketplace. Follow these
-[instructions](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
-to give the Kubeflow instance access to GCP services.
-
-> A future release will automate provisioning of KFP clusters and incorporate
-K8s Workload Identity for auth. 
+### Kubeflow Pipelines 
+Create a Kubeflow Pipelines instance on [AI Platform Pipelines](https://console.cloud.google.com/ai-platform/pipelines). 
+Once the instance is provisioned, note down the hostname (Dashboard URL).
 
 ## Cloud AI Platform Demo
 This demo uses the scikit-learn model in
 `examples/sklearn/model/sklearn_model.py` to create a training module to run on
-CAIP. First, make a copy of the scikit-learn example.
+CAIP. First, make a copy of the `sklearn` example directory.
 
 ```bash
 cp -r examples/sklearn sklearn-demo
 cd sklearn-demo
 ```
 
 Create a `config.yaml` by using the `config.yaml.example` template. See the
-[Input args](#input-args) section for details on the config parameters. Once the
+[schema](schema/CONFIG.md) for details on the config parameters. Once the
 config file is filled out, run the demo.
 
 ```bash
-python demo.py
+python3 demo.py
```

Comment:
See above comment, `python` should be set properly in the virtualenv, no need to specify `python2` vs `python3`.

Reply:
That's nice. Didn't know venv enforces the python version with which it's created. Reverted all `python3` references except for when creating the venv.